controllers/provisioners/ekscloudformation/ekscloudformation.go

```diff
@@ -488,3 +493,22 @@ func (ctx *EksCfInstanceGroupContext) processParameters() error {
 	ctx.parseTags()
 	return nil
 }
+
+//GetManagedPolicyARNs constructs managed policy arns
+func GetManagedPolicyARNs(pNames []string) string {
```

Comment:
Does this need to be exported? perhaps better as `getManagedPolicyARNs`.
Also, `policyNames` might be more readable than `pNames`

Reply:
I don't know for some reason my Test file did not detect it in GoLand so had to export it. Let me try it without exporting